views/event/schedule.pug

```diff
@@ -0,0 +1,57 @@
+extends ../layouts/width
```

Comment:
This page looks a little weird on mobile. Would it be possible to spread it out a little. I think that would fix it. It does look fantastic on the web though